Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -1338,6 +1338,7 @@ void CFrame::OnCheckNAND(wxCommandEvent&)
                                  StringFromFormat("%016" PRIx64, title_id);
       message += "\n";
     }
+    message += _("\nLaunching these titles may also fix the issues.");
```

Comment:
Perhaps it would be more beneficial to build the list beforehand so the translatable strings can be joined together like:

```cpp
void CFrame::OnCheckNAND(wxCommandEvent&)
{
  IOS::HLE::Kernel ios;
  WiiUtils::NANDCheckResult result = WiiUtils::CheckNAND(ios);
  if (!result.bad)
  {
    wxMessageBox(_("No issues have been detected."), _("NAND Check"), wxOK | wxICON_INFORMATION);
    return;
  }

  wxString message = _("The emulated NAND is damaged. System titles such as the Wii Menu and "
                       "the Wii Shop Channel may not work correctly.\n\n"
                       "Do you want to try to repair the NAND?");
  if (!result.titles_to_remove.empty())
  {
    std::string title_listing;
    Core::TitleDatabase title_db;
    for (const u64 title_id : result.titles_to_remove)
    {
      const std::string name = title_db.GetTitleName(title_id);
      title_listing += !name.empty() ? StringFromFormat("%s (%016" PRIx64 ")", name.c_str(), title_id) :
                                       StringFromFormat("%016" PRIx64, title_id);
      title_listing += "\n";
    }

    message += wxString.Format(_("\n\nWARNING: Fixing this NAND requires the deletion of titles that have "
                                 "incomplete data on the NAND, including all associated save data. "
                                 "By continuing, the following title(s) will be removed:\n\n"
                                 "%s"
                                 "\nLaunching these titles may also fix the issues.", title_listing.c_str()));

    // I forget if you'd need to use StrToWxStr on title_listing here.
  }

  ...
}
```

so it's all together and easier for translators

Reply:
Agreed, I have made this change in both UIs.